docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,114 @@
+** Attack Discovery now combines related discoveries that would previously have appeared separately
+** Attack Discovery now detects and displays an error instead of hallucinated output
+* Adds an **Install and enable** button to the **Add Elastic Rules** page, so you don't have to wait for rules to install before enabling them ({kibana-pull}191529[#191529]). 
+* Updates the Get Started tour for {elastic-sec} ({kibana-pull}192247[#192247]). 
+* Improves loading performance for various pages in {kib} ({kibana-pull}194241[#194241]). 
+* Adds Alert Suppression and Investigation Fields to the rule upgrade workflow ({kibana-pull}195499[#195499]). 
+* Adds User and Global Artifacts to the {fleet} Policy Response flyout and to the Endpoint details flyout ({kibana-pull}184125[#184125]). 
+* Allows you to recalculate entity risk scores immediately after you upload asset criticality data ({kibana-pull}187577[#187577]). 
+* Adds a {kib} advanced setting `securitySolution:maxUnassociatedNotes`, which allows you to set the maximum number of notes that can be attached to alerts and events ({kibana-pull}194947[#194947]). 
+* Adds the `IS` operator as an option when configuring a Windows signature blocklist entry ({kibana-pull}190515[#190515]). 
+* Allows you to disable the {elastic-defend} hardware call stacks feature ({kibana-pull}190553[#190553]). 
+* Improves network previews in the alert details flyout ({kibana-pull}190560[#190560]). 
+* Enhances {elastic-defend} by improving the `call_stack_final_user_module` attribution where potential `proxy_call` modules are encountered during Windows call stack analysis.
+* Adds new fields to {elastic-defend} API events to improve context for the triage of Behavior Alerts. The new `call_stack_final_user_module` fields are `allocation_private_bytes`, `protection`, `protection_provenance_path`, and `reason`.
+* Adds a new {elastic-defend} API event for https://learn.microsoft.com/en-us/windows/win32/api/ioapiset/nf-ioapiset-deviceiocontrol[`DeviceIoControl`] calls to support the detection of driver abuse. This feature is only supported on Windows 11 Desktop versions.
+* Ensures security artifacts are updated when the {elastic-defend} service starts.
+* Improves error messages that are returned when {elastic-defend} receives invalid or unsupported cryptographic keys from the {elastic-defend} policy. 
+* Ensures that {elastic-defend} tells {fleet} that it's `orphaned` if the connection between {elastic-defend} and {agent} stops for an extended period of time. {fleet} uses this information to provide you with additional troubleshooting context.
+* Adds SOCKS5 proxy support to {elastic-defend}'s {ls} output.
+* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables detection of certain defense evasions.
```

Comment:
```suggestion
* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables detection of certain defense evasions.  Users can disable this feature in {elastic-defned} <<adv-policy-settings,advanced policy>> ({kibana-pull}190553[#190553]).
```

Reply:
Making some minor tweaks to the revision:

```suggestion
* Ensures that on Windows, {elastic-defend} uses https://www.elastic.co/security-labs/finding-truth-in-the-shadows[Intel CET and AMD Shadow Stacks] to collect call stacks, where supported. This improves performance and enables the detection of certain defense evasions. You can turn this feature off in {elastic-defend} <<adv-policy-settings,advanced policy settings>> ({kibana-pull}190553[#190553]).
```